samples/ScheduleWebApp/Orchestrations/CacheClearingOrchestrator.cs

```diff
@@ -14,8 +15,12 @@ public override async Task<string> RunAsync(TaskOrchestrationContext context, st
         {
             logger.LogInformation("Starting CacheClearingOrchestration for schedule ID: {ScheduleId}", scheduleId);
 
-            // Simulate cache clearing
-            await Task.Delay(TimeSpan.FromSeconds(5));
+            TaskOptions options = new TaskOptions(tags: new Dictionary<string, string>
+            {
+                { "scheduleId", scheduleId }
+            });
+
+            await context.CallActivityAsync(nameof(CacheClearingActivity), options);
```

Comment:
I am confused what is this scheduleId used for when calling CacheClearingActivity? Can you add some comments to describe about this test? like how this tag is used.